### PR TITLE
Doc: Terminology update and minor grammar corrections

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Update "source of truth" to "system of record"
minor grammar fixes

### CODE REVIEW

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Imagine being asked to configure a **layer 3 leaf spine (L3LS)** network from sc
 ![Figure: Arista Leaf Spine topology](../_media/getting-started/single-dc-topology-physical.png)
 
 Traditionally one would configure the switches manually using a laptop, console cable, and USB key to load the correct EOS software
-image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and prone to errors. Consider what would happen if a new pair of leaf switches or new VRFs/VLANs are added to the network - changes would need to be made to most of the devices.
+image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and prone to errors. Consider what would happen if a new pair of leaf switches or new VRFs/VLANs were added to the network - changes would need to be made to most of the devices.
```
Comment (ankudinov) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]: I'd simplify:
Imagine a new pair of leaf switches or new VRFs/VLANs were added to the network - changes would need to be made to most of the devices.

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ Imagine being asked to configure a **layer 3 leaf spine (L3LS)** network from sc
...
 
-AVD is also based on best practice from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience in deploying large scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, and network topology overview as well as real-time streaming telemetry including flow-based data and more.
+AVD is also based on best practices from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience in deploying large-scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, and network topology overview as well as real-time streaming telemetry including flow-based data and more.
```
Comment (ankudinov) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]: Delete `also`

**Review by ankudinov — DISMISSED** [review]
LGTM apart from the comments above

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 3]
```diff
@@ -358,15 +358,15 @@ Running the play described above would result in several files containing the co
 
 While this play shows both the `eos_designs` and `eos_cli_config_gen` roles used together, it's entirely possible to make use of just `eos_cli_config_gen` by itself - this would allow (for example) generation of management configuration that could potentially be merged into an existing network as discussed earlier.
 
-## Source of Truth
+## System or Record
```
Comment (noredistribution) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
## System of Record
```

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md (hunk 2 above)
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
AVD is based on best practices from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience in deploying large-scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, and network topology overview as well as real-time streaming telemetry including flow-based data and more.
```

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ Ansible can run on almost anything, but in production scenarios, Ansible is typi
 Arista and accepts third-party contributions on GitHub at [aristanetworks/ansible-avd](https://github.com/aristanetworks/ansible-avd).
 
 While Ansible is the core automation engine, AVD is an Ansible Collection described above. It provides roles, modules, and plugins
-that allows the user to generate and deploy best-practice configurations to a layer three leaf-spine network.
+that allow the user to generate and deploy best-practice configurations to a layer three leaf-spine network.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]: AVD supports more than l3ls now, so maybe the end of this sentence could be changed to "... configurations to Arista based networks of various different designs."

Comment (carlbuchmann) [reply ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]: addressed in 6e84056

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md (hunk 1 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]: addressed in 6e84056

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 5]
```diff
@@ -12,16 +12,15 @@ Imagine being asked to configure a **layer 3 leaf spine (L3LS)** network from sc
...
 
-Traditionally one would configure the switches manually using a laptop, console cable, and USB key to load the correct EOS software
-image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and prone to errors. Consider what would happen if a new pair of leaf switches or new VRFs/VLANs are added to the network - changes would need to be made to most of the devices.
+Traditionally one would configure the switches manually using a laptop, console cable, and USB key to load the correct EOS software image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and error-prone. Imagine a new pair of leaf switches or new VRFs/VLANs were added to the network, changes would need to be made to most of the devices.
```
Comment (JulioPDX) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
Traditionally, one would configure the switches manually using a laptop, console cable, and USB key to load the correct EOS software image and configuration. Of course, the configuration would be manually generated using snippets from the relevant Arista design guides or copied/pasted from existing devices. Configurations must be manually adapted to every device, which is time-consuming and error-prone. Imagine a new pair of leaf switches or new VRFs/VLANs were added to the network; changes would need to be made to most of the devices.
```

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 6]
```diff
@@ -12,16 +12,15 @@ Imagine being asked to configure a **layer 3 leaf spine (L3LS)** network from sc
...
 
-AVD is also based on best practice from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience in deploying large scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, and network topology overview as well as real-time streaming telemetry including flow-based data and more.
+AVD is based on best practices from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience in deploying large-scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, and network topology overview as well as real-time streaming telemetry including flow-based data and more.
```
Comment (JulioPDX) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
AVD is based on best practices from [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides), meaning you get the full benefit of Arista's experience deploying large-scale leaf spine fabrics. AVD fully integrates with Arista CloudVision, which adds a change control workflow, continuous compliance check, network topology overview, real-time streaming telemetry including flow-based data, and more.
```

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 7]
```diff
@@ -358,15 +357,15 @@ Running the play described above would result in several files containing the co
...
+This system of record means you have a complete overview of your entire designed network configuration without having to look at individual network devices. An additional benefit of AVD is that by design, you always have a backup of your network configuration. Including automated documentation in markdown format, you no longer have to remember to update documentation about which interface is connected to which device, etc., whenever you change something on a device. Instead, it's all done automatically based on the configuration built and applied to the network devices by Ansible every time you execute the playbooks.
```
Comment (JulioPDX) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
This system of record means you have a complete overview of your designed network configuration without looking at individual network devices. An additional benefit of AVD is that by design, you always have a network configuration backup. Including automated documentation in markdown format, you no longer have to remember to update documentation about which interface is connected to which device, etc., whenever you change something on a device. Instead, it's all done automatically based on the configuration built and applied to the network devices by Ansible every time you execute the playbooks.
```

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 8]
```diff
@@ -474,7 +473,7 @@ This quickly identifies any issues that we could otherwise overlook. For example
 
 ## Day 2 Operations
 
-Day 2 operations describe most, if not all, network configuration changes that occur after the initial configuration.
+Day 2 operations describe most if not all, network configuration changes that occur after the initial configuration.
```
Comment (JulioPDX) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md]:
```suggestion
Day 2 operations describe most, if not all, network configuration changes that occur after the initial configuration.
```

**Review by chetryan — APPROVED** [review]
LGTM

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. I would do `system of record` -> `System of Record`

**Pull request merged**